pkg/drivers/kic/oci/network_create.go

```diff
@@ -38,14 +38,30 @@ const firstSubnetAddr = "192.168.49.0"
 const defaultSubnetMask = 24
 
 // name of the default bridge network, used to lookup the MTU (see #9528)
-const defaultBridgeName = "bridge"
+const dockerDefaultBridge = "bridge"
+
+// name of the default bridge network
+const podmanDefaultBridge = "podman"
 
 // CreateNetwork creates a network returns gateway and error, minikube creates one network per cluster
 func CreateNetwork(ociBin string, name string) (net.IP, error) {
 	return createDockerNetwork(ociBin, name)
 }
 
 func createDockerNetwork(ociBin string, clusterName string) (net.IP, error) {
```

Comment:
the name here seems odd, createDockerNetwork creates both docker and podman, maybe move the implementaiton to the public func ?

Reply:
Yeah, no effort has been made to clean up the function names